docs/providers/aws/guide/packaging.md

```diff
@@ -48,26 +56,28 @@ exclude:
 
 ## Artifact
 
-For complete control over the packaging process you can specify your own zip file for your service. Serverless won't zip your service if this is configured so `exclude` will be ignored.
+For complete control over the packaging process you can specify your own zip file for your service. Serverless won't zip your service if this is configured so `exclude` and `include` will be ignored.
 
 ## Example
 
-```yaml
+```yml
 service: my-service
 package:
   exclude:
     - tmp/**
     - .git/**
+  include:
+    - some-file
   artifact: path/to/my-artifact.zip
 ```
 
 ## Packaging functions separately
 
 If you want even more controls over your functions for deployment you can configure them to be packaged independently. This allows you more control for optimizing your deployment. To enable individual packaging set `individually` to true in the service wide packaging settings.
 
-Then for every function you can use the same `exclude / artifact` config options as you can service wide. The `exclude` option will be merged with the service wide options to create one `exclude` config per function during packaging.
+Then for every function you can use the same `exclude`, `include` or `artifact` config options as you can service wide. The `exclude` and `include` option will be merged with the service wide options to create one `exclude` and `include` config per function during packaging.
 
```

Comment:
@flomotlik Just discussed it with @pmuens. We are both not sure if this is a feature or a bug. Leadership needed 🚓 🚨